content/sensu-go/5.11/reference/assets.md

```diff
@@ -288,6 +293,85 @@ spec:
 
 {{< /language-toggle >}}
 
+### Example Asset with a Check
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+---
+type: Asset
+api_version: core/v2
+metadata:
+  name: sensu-prometheus-collector_linux_amd64
+spec:
+  url: https://assets.bonsai.sensu.io/ef812286f59de36a40e51178024b81c69666e1b7/sensu-prometheus-collector_1.1.6_linux_amd64.tar.gz
+  sha512: a70056ca02662fbf2999460f6be93f174c7e09c5a8b12efc7cc42ce1ccb5570ee0f328a2dd8223f506df3b5972f7f521728f7bdd6abf9f6ca2234d690aeb3808
+  filters:
+  - entity.system.os == 'linux'
+  - entity.system.arch == 'amd64'
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: prometheus_collector
+  namespace: default
+spec:
+  command: "sensu-prometheus-collector -prom-url http://localhost:9090 -prom-query up"
+  handlers:
+  - influxdb
```

Comment:
If I recall correctly, the best practice is to put this under `output_metric_handlers` instead of `handlers`.

Reply:
Yup! Nice catch, that is true. I'll get that fixed.